### PR TITLE
[uss_qualifer] Add data validation scenario to ASTM F3548-21 suite

### PR DESCRIPTION
**Pull request by punamverma**
Added data validation scenario to f3548_21 suite.

### CODE REVIEW

**Comment by punamverma** [comment]
@BenjaminPelletier PR is ready for your review. Thanks.

File: monitoring/uss_qualifier/configurations/dev/f3548_self_contained.yaml [hunk 1]
```diff
@@ -71,6 +73,14 @@ v1:
             file:
               path: test_data.che.flight_intents.invalid_flight_intents
 
+        # Details of non-conflicting flights (used in data validation scenario)
+        non_conflicting_flights:
+          resource_type: resources.flight_planning.FlightIntentsResource
+          specification:
+            planning_time: '0:05:00'
```
Comment (BenjaminPelletier) [inline monitoring/uss_qualifier/configurations/dev/f3548_self_contained.yaml:80]: nit: This field doesn't exist in FlightIntentsSpecification any more and can be deleted (like the others in this file which weren't caught previously).

(note: "nit" means the reviewer is nit-picking and the comment doesn't have to be addressed in order to merge the PR)

File: monitoring/uss_qualifier/configurations/dev/f3548_self_contained.yaml [hunk 2]
```diff
@@ -80,6 +90,14 @@ v1:
             # A USS that hosts a DSS instance is also a participant in the test, even if they don't fulfill any other roles
             participant_id: uss1
             base_url: http://dss.uss1.localutm
+        mock_uss:
```
Comment (BenjaminPelletier) [inline monitoring/uss_qualifier/configurations/dev/f3548_self_contained.yaml:93]: In keeping with the rest of this file, there should be a comment explaining what this resource is -- we use this configuration as a "trainer" example when someone wants to create their own configuration, so it's important that its content be understandable for someone without other experience with configurations.

**Review by BenjaminPelletier — APPROVED** [review]
Note: Approval with comments means the PR should be good to merge after comments have been addressed (except nits don't need to be addressed; they can simply be acknowledged) -- just one non-nit comment here.

File: monitoring/uss_qualifier/configurations/dev/f3548_self_contained.yaml (hunk 1 above)
Comment (punamverma) [reply monitoring/uss_qualifier/configurations/dev/f3548_self_contained.yaml:80]: Ok, will remove planning_time in the next PR.

File: monitoring/uss_qualifier/configurations/dev/f3548_self_contained.yaml (hunk 2 above)
Comment (punamverma) [reply monitoring/uss_qualifier/configurations/dev/f3548_self_contained.yaml:95]: Added comment.

**Comment by punamverma** [comment]
@BenjaminPelletier Acknowledged the comments. Added the missing comment.

**Pull request merged**